src/humidity.toit

```diff
@@ -6,24 +6,23 @@ import system.services
 
 import .clients.humidity as clients
 
-SELECTOR ::= services.ServiceSelector
+SELECTOR-v1 ::= services.ServiceSelector
     --uuid="6e8ad988-ea25-4e65-bca3-3e3ee1762b85"
```

Comment:
Maybe pull the UUID out in a private constant. It should not change between versions.

Reply:
done.